File: LeetCode/Easy/0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        Map<Integer, Integer> map=new LinkedHashMap<>();
        for(int num : nums) {
            if(!map.containsKey(num)) {
                map.put(num, 1);
            }
        }
        System.out.println(map);
        int index=0;
        for(Map.Entry<Integer, Integer> entry : map.entrySet()) {
            nums[index++]=entry.getKey();
        }
        return index;
    }
}

File: LeetCode/Easy/0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int count=0;
        vector <int> res;
        for(int i=0;i<nums.size();i++){
            res.push_back(nums[i]);
        }
        while(nums.size()>0){
            nums.pop_back();
        }
        for(int i=0;i<res.size();i++){
            int j;
            for(j=0;j<res.size();j++){
                if(res[i]==res[j]){
                    break;
                }
            }
            if(i==j){
                nums.push_back(res[i]);
                count++;
            }
        }
        return count;
    }
};